src/components/editMeasure/populationCriteria/populationCriteriaSideNav/PopulationCriteriaSideNav.tsx

```diff
@@ -197,13 +225,12 @@ export default function PopulationCriteriaSideNav(
                               type="C"
                               orientation="vertical"
                               id={index}
+                              label={linkInfo.title}
                               data-testid={linkInfo.dataTestId}
-                              label={
```

Comment:
what was the issue with this?

Reply:
@adongare The issue with this was the group number was messed up when using the common component, so every criteria a user clicks, it was all pointing to group 1.